WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -137,7 +137,8 @@ + (NSArray *)routes
     return FBResponseWithErrorFormat(@"Missing 'value' parameter");
   }
   if (element.elementType == XCUIElementTypePickerWheel) {
-    [element adjustToPickerWheelValue:value];
+    NSString *strValue = [value componentsJoinedByString:@""];
```

Comment:
Can you check class and depending on class (String or Array) do what is appropriate? So that it can handle both?
After we migrate client I'll remove this check.